src/index.ts

```diff
@@ -96,14 +95,14 @@ export * from './identity/configuration/DevIdPConfigurationFactory';
 export * from './identity/storage/InMemoryIdPStorageAdapter';
 export * from './identity/storage/ResourceStoreStorageAdapter';
 
-// SolidIdentityProvider/InteractionPolicies
-export * from './identity/interaction/EmailPasswordInteractionPolicyHttpHandler';
-export * from './identity/interaction/handler/AbortInteractionHttpHandler';
-export * from './identity/interaction/handler/BaseInteractionHttpHandler';
-export * from './identity/interaction/handler/ConfirmInteractionHttpHandler';
-export * from './identity/interaction/handler/ContinueInteractionHttpHandler';
-export * from './identity/interaction/handler/GetInteractionHttpHandler';
-export * from './identity/interaction/handler/LoginInteractionHttpHandler';
+// SolidIdentityProvider/InteractionPolicies/EmailPassword
+export * from './identity/interaction/email-password/EmailPasswordInteractionPolicyHttpHandler';
+export * from './identity/interaction/email-password/handler/InitialInteractionChooser';
+export * from './identity/interaction/email-password/handler/OidcInteractionCompleter';
+export * from './identity/interaction/email-password/handler/RegisterHanlder';
+export * from './identity/interaction/email-password/handler/ResetPasswordHandler';
+export * from './identity/interaction/email-password/handler/ForgotPasswordHandler';
+export * from './identity/interaction/email-password/handler/LoginHandler';
```

Comment:
Yes, potentially they could, and when it comes time to build a second interaction policy, I think some slight refactoring would be in order. But, for now, I don't think the architectural effort to anticipate that is worth it. The architecture is already flexible so it shouldn't be too hard to change things when a second one come along.

Reply:
The `email-password` directory was removed mostly for that reason. I don't think it makes sense to have it. You can have multiple interaction handlers and they build from a pool of handlers.

 I have to admit I'm also not a fan of deep folder structures, I think it would be good to restore as it was previously.